wx-core/src/generated.rs

```diff
@@ -434,14 +436,22 @@ impl<const OWNED: bool> CheckListBoxIsOwned<OWNED> {
     }
 }
 // Mix-in(s) to wxCheckListBox
-impl<const OWNED: bool> ItemContainerMethods for CheckListBoxIsOwned<OWNED> {
-    fn as_item_container(&self) -> *mut c_void {
-        unsafe { ffi::wxCheckListBox_AsItemContainer(self.as_ptr()) }
+impl<const OWNED: bool> AsRef<ItemContainer> for CheckListBoxIsOwned<OWNED> {
+    fn as_ref(&self) -> &ItemContainer {
+        unsafe {
+            let p = &ffi::wxCheckListBox_AsItemContainer(self.as_ptr()) as *const *mut c_void
```

Comment:
Taking pointer to temporary value. This is danger because it doen't outlive this expression.

This program seems to work but, it is occasional.